test/test_readers_xls_reader.rb

```diff
@@ -45,9 +45,10 @@ def test_xls_excel_standardized_open
     def test_txt_in_xls_open
       w = Workbook::Book.new
       w.import(File.join(File.dirname(__FILE__), "artifacts/txt_in_xls.xls"))
-      assert_equal([:naam,:nummer,:ilt,:corporate_key,:naam_medewerker, nil, nil, :telefoon, :openingsdatum],w.sheet.table.header.to_symbols)
-      assert_equal(["dddd",2222,"i9000","asd","Anita",nil,"Betera","012-3456789",Date.new(2011,10,5)],w.sheet.table[1].collect{|a| a.value})
+      assert_equal([:naam, :nummer, :ilt, :corporate_key, :naam_medewerker, nil, nil, :telefoon, :openingsdatum], w.sheet.table.header.to_symbols)
+      assert_equal(["dddd", 2222, "i9000", "asd", "Anita", nil, "Betera", "012-3456789", '5-10-2011'], w.sheet.table[1].collect{|a| a.value})
```

Comment:
Have to check this one:
a) there used to be a time that specifically formatted csv files got an xls extension just to make them import in excel easily (see e.g. http://www.ebrueggeman.com/blog/generating-php-comma-csv-and-tab-delimited-files )
b) 5-10-2011 is a popular non-US date notation; but Workbook needs a better way of dealing with date parsing